pex/testing.py

```diff
@@ -525,20 +533,18 @@ def run_pyenv(args):
     return interpreter_location, python, pip, run_pyenv
 
 
-def ensure_python_venv(version, latest_pip=True, system_site_packages=False):
+def ensure_python_venv(
+    version,  # type: str
+    latest_pip=True,  # type: bool
+    system_site_packages=False,  # type: bool
+):
+    # type: (...) -> Tuple[str, str]
     _, python, pip, _ = ensure_python_distribution(version)
     venv = safe_mkdtemp()
-    if version in _ALL_PY3_VERSIONS:
-        args = [python, "-m", "venv", venv]
-        if system_site_packages:
-            args.append("--system-site-packages")
-        subprocess.check_call(args=args)
-    else:
-        subprocess.check_call(args=[pip, "install", "virtualenv==16.7.10"])
-        args = [python, "-m", "virtualenv", venv, "-q"]
-        if system_site_packages:
-            args.append("--system-site-packages")
-        subprocess.check_call(args=args)
```

Comment:
Does removing this mean python27 doesn't have venv support?

Reply:
Nope, it still does, but the Python 2.7 has to be found on your machine pre-installed. That's these additions: https://github.com/pantsbuild/pex/pull/1914/commits/24ca825fa5f59e2564d393c1dbb88588138bcc2f#diff-79ebd789ddd6317a85b1ccbcddcbdefc0ce8b07ed30066d9bc58805d1c22f83bR596-R622